# Generated by Django 2.2.1 on 2019-05-21 10:10

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('contact', '0002_auto_20190514_1517'),
    ]

    operations = [
        migrations.AddField(
            model_name='contactcontent',
            name='email_error_message',
            field=models.CharField(default="Erreur lors de l'envoi du message. Réessayez plus tard ou contactez nous par email ou par téléphone.", max_length=250, verbose_name="Message affiché en cas d'échec à la soumission du formulaire de contact"),
            preserve_default=False,
        ),
        migrations.AddField(
            model_name='contactcontent',
            name='email_success_message',
            field=models.CharField(default='Message envoyé! Nous vous répondrons dans les plus brefs délais.', max_length=250, verbose_name='Message affiché en cas de succès à la soumission du formulaire de contact'),
            preserve_default=False,
        ),
    ]
